README.md

```diff
@@ -200,11 +209,146 @@ df <- read.df(table_path, "deltaSharing")
+val df = spark.read.format("deltaSharing")
+  .option("readChangeFeed", "true")
+  .option("startingVersion", "3")
+  .load(tablePath)
+```
+
+### Streaming
+Starting from release 0.6.0, delta sharing table can be used as the data source for [spark structured streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html).
+Once the provider shares a table with history, the recipient can perform a streaming query on the table.
+```scala
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+val df = spark.readStream.format("deltaSharing")
+  .option("startingVersion", "1")
+  .option("ignoreChanges", "true")
+  .load(tablePath)
+```
+
 ### Table paths
 
 - A profile file path can be any URL supported by Hadoop FileSystem (such as `s3a://my_bucket/my/profile/file`).
 - A table path is the profile file path following with `#` and the fully qualified name of a table (`<share-name>.<schema-name>.<table-name>`).
 
+# Other Connectors
+<table>
+<tr>
+<th>Connector</th>
+<th>Link</th>
+<th>Status</th>
+<th>Supported Features</th>
+</tr>
+<tr>
+<td>Power BI</td>
+<td>Databricks owned</td>
+<td>Released</td>
+<td>QueryTableVersion<br>QeuryTableMetadata<br>QueryTableLatestSnapshot</td>
+</tr>
+<tr>
+<td>Node.js</td>
+<td>
+
+[goodwillpunning/nodejs-sharing-client](https://github.com/goodwillpunning/nodejs-sharing-client)
+</td>
+<td>Released</td>
+<td>QueryTableVersion<br>QeuryTableMetadata<br>QueryTableLatestSnapshot</td>
+</tr>
+<tr>
+<td>Java</td>
+<td>
+
+[databrickslabs/delta-sharing-java-connector](https://github.com/databrickslabs/delta-sharing-java-connector)
+</td>
+<td>Released</td>
+<td>QueryTableVersion<br>QeuryTableMetadata<br>QueryTableLatestSnapshot</td>
+</tr>
+<tr>
+<td>Arcuate</td>
+<td>
+
+[databrickslabs/arcuate](https://github.com/databrickslabs/arcuate)
+</td>
+<td>Released</td>
+<td>QueryTableVersion<br>QeuryTableMetadata<br>QueryTableLatestSnapshot</td>
+</tr>
+<tr>
+<td>Rust</td>
+<td>
+
+[r3stl355/delta-sharing-rust-client](https://github.com/r3stl355/delta-sharing-rust-client)
+</td>
+<td>Released</td>
+<td>QueryTableVersion<br>QeuryTableMetadata<br>QueryTableLatestSnapshot</td>
+</tr>
+<tr>
+<td>Go</td>
+<td>
+
+[magpierre/delta-sharing](https://github.com/magpierre/delta-sharing/tree/golangdev/golang/delta_sharing_go)
+</td>
+<td>Un-released</td>
+<td>N/A</td>
+</tr>
+<tr>
+<td>C++</td>
+<td>
+
+[magpierre/delta-sharing](https://github.com/magpierre/delta-sharing/tree/cppdev/cpp/DeltaSharingClient)
+</td>
+<td>Un-released</td>
+<td>N/A</td>
+</tr>
+<tr>
+<td>Airflow</td>
+<td>
+
+[apache/airflow](https://github.com/apache/airflow/pull/22692)
+</td>
+<td>Un-released</td>
+<td>N/A</td>
+</tr>
+<tr>
+<td>Excel-Connector</td>
+<td>
+
+[https://www.exponam.com/solutions/](https://www.exponam.com/solutions/)
+</td>
+<td>limited-release</td>
+<td>N/A</td>
+</tr>
+<tr>
+<td>R</td>
+<td>TBD</td>
+<td>In-Progress</td>
+<td>N/A</td>
+</tr>
+<tr>
+<td>BigQuery</td>
```

Comment:
Maybe it's better to omit these connectors that haven't been started?

Reply:
removed.